video_worker/api_communicate.py

```diff
@@ -104,6 +109,7 @@ def determine_veda_pk(self):
         )
 
         if y.status_code != 200:
+            logger.error('VEDA API Fail: Check VEDA API config')
             ErrorObject().print_error(
                 message='VEDA API Fail: Check VEDA API config'
             )
```

Comment:
@muzaffaryousaf I have removed `ErrorObject` in for api_communicate.py and coverage has increased because we were not checking `ErrorObject` previously.